packages/node-core/src/indexer/dictionary.service.ts

```diff
@@ -165,9 +165,16 @@ export class DictionaryService implements OnApplicationShutdown {
     startBlock: number,
     queryEndBlock: number,
     batchSize: number,
+    metadataTableName: string,
```

Comment:
Can we make this the last arg and default like so
```suggestion
    metadataTableName = '_metadata',
```